packages/components/src/components/Combobox/Combobox.tsx

```diff
@@ -0,0 +1,158 @@
+import React, { useState, forwardRef, ForwardedRef } from "react";
+import map from "lodash/map";
+import filter from "lodash/filter";
+import includes from "lodash/includes";
+import toLower from "lodash/toLower";
+import isBoolean from "lodash/isBoolean";
+import isFunction from "lodash/isFunction";
+import {
+  ComboboxProps as AriakitComboboxProps,
+  useComboboxStore,
+} from "@ariakit/react";
+import find from "lodash/find";
+import { ComboboxInput } from "./ComboboxInput/ComboboxInput";
+import { ComboboxPopover } from "./ComboboxPopover/ComboboxPopover";
+import { ComboboxItem } from "./ComboboxItem/ComboboxItem";
+import { ComboboxProvider } from "./ComboboxProvider/ComboboxProvider";
+
+export type ComboboxItemProps = {
+  label: string;
+  value: string;
+  disabled?: boolean;
+};
+
+export type ComboboxProps<T extends ComboboxItemProps = ComboboxItemProps> =
```

Comment:
Because `{...props}` are being passed to the input we could pass tokens as below and that will work

``` typescript
<Combobox
      backgroundColor="bgError"
      borderColor="bgError"
      borderStyle="borderStyleSolid"
      borderWidth="borderWidth30"
      filterItem={customFilter}
      items={customItems}
      padding="d5"
/>
```

so the type here would need to be

```
export type ComboboxProps<T extends ComboboxItemProps = ComboboxItemProps> =
  BoxProps &
    Omit<AriakitComboboxProps, "store"> & {

```